charmed_openstack_info/data/lp-builder-config/ceph.yaml

```diff
@@ -4,11 +4,11 @@ defaults:
   branches:
     master:
       build-channels:
-        charmcraft: "2.x/stable"
+        charmcraft: "2.2/stable"
```

Comment:
Are you sure you want to switch to 2.2/stable rather than stay on 2.x/stable; it will mean no interim ubuntu releases (e.g. mantic, etc) and you won't get access to 'devel' either.

Reply:
Thanks for pointing out that caveat @ajkavanagh  -- indeed we do need interim releases as well. I've uploaded a new change